packages/sources/layer2-sequencer-health/README.md

```diff
@@ -0,0 +1,49 @@
+# Chainlink External Adapter for Example
+
+Adapter that checks the Layer 2 Sequencer status
+
+
+### Environment Variables
+
+| Required? |  Name   |                                                        Description                                                         | Options | Defaults to |
+| :-------: | :-----: | :------------------------------------------------------------------------------------------------------------------------: | :-----: | :---------: |
+|           | `DELTA` | Maximum time in miliseconds from last seen block for the sequencer to be considered healthy |         |     180000 (3 min)        |
```

Comment:
Do we have any data around this? I'd like to make it smaller if possible, 1 or 2 minutes.

Reply:
Really depends on the Network workload, more transactions, more blocks. Assuming that our L2 feeds will be updated quite frequently, I think we can lower it to 1 or 2